docs/setup-dev-env.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Setup Development Environment
+keywords: homepage
+sidebar: talk_sidebar
+permalink: /setup-development-environment.html
+summary:
+---
+## Talk development environment:
+* Docker for Mongo + Redis
+* Source for running Talk
+
+##  [Installation From Source]({% link install-source.md %})
+
+
+TODO: intergate this into Installation from source! 
+“But there is also a ```plugins.default.json``` you could use at the beginning.”
+
+## Running in development mode using ‘.env’ file
+In a CLI with node set to node 7.8 e.g. with nvm.
+```
+$ yarn dev-start
+Error: ENOENT: no such file or directory, open ‘.env’
+```
+creat ‘.env’ file then
+```
+$ yarn dev-start
+Error: Facebook cannot be enabled, missing one of TALK_FACEBOOK_APP_ID, TALK_FACEBOOK_APP_SECRET, TALK_ROOT_URL
+```
+edit ‘.env’ like this sample
+
+Sample .env file as in more detail documented in [Configuration]({% link configuration.md %}).
+
+```
```

Comment:
We should not include sample config in this form.

The reason we don't provide sample config files is that it inevitably ends up running in production, which I want to prevent at all costs. Instead direct the users to the configuration page to learn about the required configuration.